.travis.yml

```diff
@@ -3,9 +3,12 @@ os:
   - linux
   - osx
 julia:
+  - 1.5
```

Comment:
`1` is synonymous to the last Julia release so `1.5` in this case. I feel that is is less burden to use `1` so that we don't have to update to all Julia `1.x` release.

Reply:
Oh!  I didn't know that, I'll change it back.

Reply:
I also learned this rather recently, it seems this fact is not widely known enough ^^